Plugins/NativeServerPlugin/nvEncConfig.json

```diff
@@ -2,8 +2,8 @@
   "useSoftwareEncoding": false,
   "serverFrameCaptureFPS": 60,
   "NvencodeSettings": {
-    "bitrate": 5500000,
-    "minBitrate": 3500000,
+    "bitrate": 5000000,
+    "minBitrate": 5000000,
```

Comment:
do we want to take these changes or were these for local testing?

Reply:
Oh, it's mainly for debugging performance. Will revert.